cmd/atlas/atlas.go

```diff
@@ -99,21 +94,35 @@ func createConfigFromMongoCLIConfig() {
 	_, err = io.Copy(out, in)
 	if err != nil {
 		log.Printf("There was an error generating %s: %v", atlasConfigPath, err)
+		return
 	}
 
 	_, _ = fmt.Fprintf(os.Stderr, `AtlasCLI has copied your MongoCLI configuration to: %s
 
 `, atlasConfigPath)
 }
 
-func mongoCLIConfigPath() string {
-	configDir, err := config.MongoCLIConfigHome()
-	if err != nil {
-		return ""
+func mongoCLIConfigFile() (in *os.File, err error) {
+	configPath := ""
+	configDir := ""
+
+	if configDir, err = config.MongoCLIConfigHome(); err == nil {
+		configPath = fmt.Sprintf("%s/config.toml", configDir)
+	}
+
+	// Try to open new file
+	in, err = os.Open(configPath)
+	if err == nil {
+		return in, nil
+	}
+
+	// Try finding old file since new file opening failed/did not exist
+	if configDir, err = config.OldMongoCLIConfigHome(); err == nil {
+		configPath = fmt.Sprintf("%s/mongocli.toml", configDir)
 	}
 
-	configPath := fmt.Sprintf("%s/mongocli.toml", configDir)
-	return configPath
+	// Try to open old file
+	return os.Open(configPath)
```

Comment:
nitpick: my personal style is never to open files within functions like this rather to return the path and let the consumer open/close files, I personally believe writing code to open/close should always be together (that is one of the strengths of defer keyword).

If you are relying on open errors to fallback consider:
```go
if _, err := os.Stat("/path/to/whatever"); errors.Is(err, os.ErrNotExist) {
  // path/to/whatever does not exist
}
```